--- zc.config	2018-01-15 19:51:31.952768945 -0600
+++ zc.config.prob	2018-01-15 19:54:33.876843750 -0600
@@ -13,7 +13,7 @@
 #example: checkingStatus = PROBE_COMPRESSOR
 #example: checkingStatus = ANALYZE_DATA
 #example: chekcingStatus = COMPARE_COMPRESSOR
-checkingStatus = ANALYZE_DATA
+checkingStatus = PROBE_COMPRESSOR
 
 #two options for execution mode: either ONLINE or OFFLINE;
 #ONLINE means running with parallel application such as MPI programs to check the compression at runtime (the data are produced by simulations at runtime)
