--- Makefile.bk	2017-09-06 14:51:44.951457688 -0500
+++ Makefile.bk.p	2017-09-06 14:52:32.225855729 -0500
@@ -2,7 +2,8 @@
 ##   Makefile to test SZ library.
 
 ##   PLEASE SET THIS VARIABLE BEFORE COMPILING
-SZPATH		= /home/sdi/Install/sz-1.4.11-beta-install
+SZPATH		= ../../SZ/sz-install
+ZCPATH		= ../../Z-checker/zc-install
 ##   COMPILERS
 CC		= gcc
 FT		= gfortran
@@ -52,6 +53,12 @@
 testdouble_batch_compress:      testdouble_batch_compress.c
 	$(CC)  -g -O3 -o testdouble_batch_compress testdouble_batch_compress.c $(SZFLAG)
 
+testfloat_CompDecomp:	testfloat_CompDecomp.c
+	$(CC)  -g -O3 -o testfloat_CompDecomp testfloat_CompDecomp.c $(ZCFLAG) $(SZFLAG)
+
+testdouble_CompDecomp:	testdouble_CompDecomp.c
+	$(CC)  -g -O3 -o testdouble_CompDecomp testdouble_CompDecomp.c $(ZCFLAG) $(SZFLAG)
+
 clean:
-	rm -f test*_compress test*_decompress test*_compress_f test*_decompress_f testfloat_batch_compress testdouble_batch_compress_f
+	rm -f test*_compress test*_decompress test*_compress_f test*_decompress_f testfloat_batch_compress testdouble_batch_compress_f testfloat_CompDecomp testdouble_CompDecomp
 
