--- testdouble_CompDecomp.sh	2017-05-20 15:43:39.020897510 -0500
+++ testdouble_CompDecomp_fast.sh	2017-05-20 16:02:03.371696107 -0500
@@ -17,8 +17,8 @@
 fileList=`cd "$dataDir";ls *.dat`
 for file in $fileList
 do
-        echo testdouble_CompDecomp sz.config zc.config "sz($absErrBound)" "$file" $absErrBound "$dataDir/$file" $dim1 $dim2 $dim3 $dim4
-        ./testdouble_CompDecomp sz.config zc.config "sz($absErrBound)" "$file" $absErrBound "$dataDir/$file" $dim1 $dim2 $dim3 $dim4
+        echo testdouble_CompDecomp sz.config zc.config "sz_f($absErrBound)" "$file" $absErrBound "$dataDir/$file" $dim1 $dim2 $dim3 $dim4
+        ./testdouble_CompDecomp sz.config zc.config "sz_f($absErrBound)" "$file" $absErrBound "$dataDir/$file" $dim1 $dim2 $dim3 $dim4
 done
 
 echo "complete"
